aspnetcore/signalr/configuration.md

```diff
@@ -447,7 +447,7 @@ The following table describes options for configuring SignalR hubs:
 | `SupportedProtocols` | All installed protocols | Protocols supported by this hub. By default, all protocols registered on the server are allowed. Protocols can be removed from this list to disable specific protocols for individual hubs. |
 | `EnableDetailedErrors` | `false` | If `true`, detailed exception messages are returned to clients when an exception is thrown in a Hub method. The default is `false` because these exception messages can contain sensitive information. |
 | `StreamBufferCapacity` | `10` | The maximum number of items that can be buffered for client upload streams. If this limit is reached, the processing of invocations is blocked until the server processes stream items.|
-| `MaximumReceiveMessageSize` | 32 KB | Maximum size of a single incoming hub message. |
```

Comment:
nit: the wording "We recommend setting" makes it sound like the user /should/ change the value.
"We recommend keeping the value small", "Increasing this value may increase the risk of DOS...", etc.

Reply:
The 2nd one sounds good :ear: to me. I'll try that on the next commit.